Do not try to find ragel. It would only be used to convert
request_parser.rl into request_parser.cpp and we have no need to do
so since request_parser.cpp is already in the source code tarball.
--- CMakeLists.txt.orig	2019-08-05 11:13:37.000000000 -0500
+++ CMakeLists.txt	2022-11-16 23:29:01.000000000 -0600
@@ -72,8 +72,6 @@
 INCLUDE_DIRECTORIES (${Boost_INCLUDE_DIRS})
 LINK_DIRECTORIES (${Boost_LIBRARY_DIRS})
 
-INCLUDE (FindRAGEL)
-FIND_PACKAGE (RAGEL)
 
 FIND_PACKAGE (Threads)
 
